components/table/src/cell.ts

```diff
@@ -32,17 +33,15 @@ import {
   SkipSelf,
   ViewEncapsulation,
 } from '@angular/core';
-import { Subject, Subscription, merge } from 'rxjs';
-import { filter, startWith, switchMap, takeUntil } from 'rxjs/operators';
-
 import {
   DtIndicator,
-  _addCssClass,
   isDefined,
+  _addCssClass,
   _parseCssValue,
   _removeCssClass,
 } from '@dynatrace/barista-components/core';
-
+import { merge, Subject, Subscription } from 'rxjs';
+import { filter, startWith, switchMap, takeUntil } from 'rxjs/operators';
```

Comment:
Unnecessary sorting change.

Reply:
We have the tsconfig `"noUnusedParameters": true` in place that breaks all-time my build if there is an unused import. I don't fix them all time by hand –  I use the organize imports for that so sorry for the changes.